Core/examples/Main.cpp

```diff
@@ -133,7 +142,7 @@ int main()
 
     buildSceneGraph(glState);
 
-    auto[width, height, channels, data] = readTexture("examples/textures/image.png");
+    auto [width, height, channels, data] = readTexture("examples/textures/image.png");
```

Comment:
Did you update visual studio or clang-format? this looks better, but i remember clang-format previously wanted the other